internal/lpastore/lpa.go

```diff
@@ -495,7 +495,7 @@ func FromDonorProvidedDetails(l *donordata.Provided) *lpadata.Lpa {
 		Restrictions:                             l.Restrictions,
 		WhenCanTheLpaBeUsed:                      l.WhenCanTheLpaBeUsed,
 		LifeSustainingTreatmentOption:            l.LifeSustainingTreatmentOption,
-		SignedAt:                                 l.SignedAt,
+		SignedAt:                                 l.WitnessedByCertificateProviderAt,
```

Comment:
I understand this change but it bothers me that there's some potential for a drift between signing and witnessing dates here if a CP fails to witness after the donor signs that could be confusing. To keep in sync, do we need to add `WitnessedByCertificateProviderAt` to the Lpa store data? (not as part of this ticket)

Reply:
potentially, but we'll never drive logic off our `SignedAt` date as it isn't _really_ signed at that point